loopring/config.py

```diff
@@ -11,12 +11,7 @@
 
 # Do the mpc for the following circuits
 circuits = [ \
-    [0, True, [1, 2, 4, 8, 14, 31, 63, 128, 256, 512, 1024]], \
-    [1, False, [1, 2, 4, 8, 15, 31, 64, 128, 256]], \
-    [2, False, [1, 2, 4, 8, 16, 32, 64, 128, 256]], \
-    [3, True, [1, 2, 4, 8, 16, 32, 64, 128, 256]], \
-    [4, True, [1, 2, 4, 8, 16, 32, 64, 128, 256]], \
-    [5, True, [1, 2, 4, 8, 28, 58, 120, 243]], \
+    [0, False, [16, 64, 128, 192, 256, 320, 384, 448, 512]], \
```

Comment:
Most optimal values will be where the number of constraints are just below a 2^n (because of the FFT), that's why we had those specific values here previously. To keep the number of constraints below 2^26, the max block size will probably be around 480 transactions/block. To find these optimal block sizes I generated the blocks and inspected the number of constraints. Because the audit may still change some circuit code, it's probably a bit too soon now to do this though.